### PR TITLE
SLVUU-97 add workspace settings for default formatter

### PR DESCRIPTION
**Pull request by vferraro-scottlogic**
### Changes
- include formatting settings in vscode/settings.json
- add extensions.json with recommended extensions

### CODE REVIEW

**Comment by cfisher-scottlogic** [comment]
Are you updating the docs in a separate PR?

File: README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ Then, back in vuu-ui, run the launch script"
 npm run launch:demo:electron
 ```
 
+### Workspace Settings 
+A set of VSCode settings is shared as workspace settings. In order to use the workspace settings the project needs to be opened in VSCode from vuu-ui/ folder 
+
```
Comment (cfisher-scottlogic) [inline README.md:69]: I would expand this to mention what it is the VSCode workspace settings is providing so other IDE users can match VSCode. I'd also mention the intelliJ/Java settings as well.

**Comment by cfisher-scottlogic** [comment]
> > Are you updating the docs in a separate PR?
> 
> I have updated only the README as part of this task. I think this will be included in the docusaurus docs [when we rework the docusaurus and the README](https://github.com/orgs/ScottLogic/projects/21/views/1?filterQuery=&pane=issue&itemId=44427245) (similarly to the layout server documentation)

Ah, I completely didn't spot the README change!

File: README.md (hunk 1 above)
Comment (vferraro-scottlogic) [reply README.md]: I have added the list of settings provided by the workspace. I don't think the intelliJ/Java settings should be in scope for this change

Comment (cfisher-scottlogic) [reply README.md]: How come?

Comment (vferraro-scottlogic) [reply README.md]: I have added IntelliJ Settings for Java as shown in the screenshot on the ticket

**Review by cfisher-scottlogic — APPROVED** [review]

**Review by pling-scottlogic — APPROVED** [review]

**Comment by vferraro-scottlogic** [comment]
[raised PR to finos](https://github.com/finos/vuu/pull/1077)

**Pull request closed without merging**